Tests/test_lib_pack.py

```diff
@@ -151,10 +151,6 @@ def test_YCbCr(self):
         self.assert_pack("YCbCr", "YCbCr", 3, (1, 2, 3), (4, 5, 6), (7, 8, 9))
         self.assert_pack("YCbCr", "YCbCr;L", 3,
                          (1, 4, 7), (2, 5, 8), (3, 6, 9))
-        self.assert_pack(
```

Comment:
I change my mind. This is actually breaking change, because 'YCbCrX' is allowed for using in user-space:

```python
data = im.tobytes("raw", "YCbCrX")
Image.frombytes(im.mode, im.size, data, "raw", "YCbCrX", 0, 1)
```

I think it's better to leave YCbCrX mode and drop modes with more Xs

Reply:
added them back.